feat: redundant storage (#13)
WalshyDev/cf-status-worker: README.md

  4b. (optional) If you want publishing, you'll also need to add a Discord bot token with `wrangler secret put DISCORD_TOKEN`
5. Run `npm run publish` :)

## Data redundancy

> This worker is using KV as its main store, but that can go down, so you can additionally
> save data to R2 or D1.

Reviewer: ```suggestion
This worker is using KV as its main store, but if that was to suffer a problem, you can additionally
save data to R2 or D1.
```

```toml
[[d1_databases]]
binding = "D1"
database_name = ""
database_id = ""

[[r2_buckets]]
binding = "R2"
bucket_name = ""
```

To setup D1, you need to run the following SQL statement:

> `CREATE TABLE IF NOT EXISTS KV (key TEXT UNIQUE, value TEXT)`

Reviewer: ```suggestion
`CREATE TABLE IF NOT EXISTS incidents (key TEXT UNIQUE, value JSON)`
```

Author: This is SQLite, JSON will actually be mapped to NUMERIC.
See [datatype3 3.1](https://www.sqlite.org/datatype3.html#determination_of_column_affinity):
1. If the declared type contains the string "INT" then it is assigned INTEGER affinity.
2. If the declared type of the column contains any of the strings "CHAR", "CLOB", or "TEXT" then that column has TEXT affinity. Notice that the type VARCHAR contains the string "CHAR" and is thus assigned TEXT affinity.
3. If the declared type for a column contains the string "BLOB" or if no type is specified then the column has affinity BLOB.
4. If the declared type for a column contains any of the strings "REAL", "FLOA", or "DOUB" then the column has REAL affinity.
5. Otherwise, the affinity is NUMERIC.
